### PR TITLE
Add VBA loading animation

### PR DESCRIPTION
**Pull request by mountiny**
cc @MitchExpensify 

### Details

Adds an animation in between VBA steps to make the UI more intuitive.

### Fixed Issues

$ https://github.com/Expensify/Expensify/issues/172797

### Tests

1. Create a new workspace.
2. Navigate to it and click `Get Started`
3. Choose `Connect Manually`
4. Fill in the Routing number and Account number and click `Save & Continue`
5. Fill in all the company information and click `Save & Continue`
6. Make sure the loading page/animation looks like on the videos

Additionally, can you please test on physical Android device, if the gif animation really works between the steps of VBA flow? In emulator, it has been static image, but the entire app is very slow in my emulator so not sure what has caused it.

### QA Steps

Same as tests.

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android - only emulator

#### Web

https://user-images.githubusercontent.com/36083550/128869377-da473484-c2a8-4031-9bae-a12e6f028acf.mov

#### Mobile Web

https://user-images.githubusercontent.com/36083550/128868297-3edf5cf7-020d-4d5a-bba5-1fff6419f26f.mov

#### Desktop

https://user-images.githubusercontent.com/36083550/128868268-b6d25d45-4216-40a8-b34e-694f8fd8c259.mov

#### iOS

https://user-images.githubusercontent.com/36083550/128868320-3f6e4b8d-cb39-4cd0-9e3d-39b5b2fc3e8b.mov

#### Android

https://user-images.githubusercontent.com/36083550/128873902-36663723-0121-4402-a6ab-14e3905710c7.mov

### CODE REVIEW

**Comment by mountiny** [comment]
I will need to follow [these steps](https://stackoverflow.com/questions/35594783/how-do-i-display-an-animated-gif-in-react-native) to make sure the gif works well on Android too.

i will try to update this over weekend. I will be ooo next Monday.

**Comment by mountiny** [comment]
@TomatoToaster This should be ready for review!

**Comment by mountiny** [comment]
I have not been able to test it on Android, my setup has gone wild and can't run it, but that is not related to the changes here.

**Comment by mountiny** [comment]
@TomatoToaster Alright, I have managed to make my android emulator run it by uninstalling the app from the emulator and then running it again.

It is extremely sluggish which is as always I guess but then the gif of the animation does not move at all. So I do not know if that is just caused by the sluggishness of the system or the gif cant be animated on Android. Are you able to test it on physical device? I do not have any :/ 

Thank you for the review!

File: android/app/src/main/java/com/expensify/chat/generated/BasePackageList.java
Comment (TomatoToaster) [inline android/app/src/main/java/com/expensify/chat/generated/BasePackageList.java:13]: Is this file meant to be included? I know I see these files sometimes generated when I run the android emulator, I'm not  sure if they're meant to be committed. They are a bit of a mystery to me.

Comment (TomatoToaster) [reply android/app/src/main/java/com/expensify/chat/generated/BasePackageList.java:13]: I've asked about it here: https://expensify.slack.com/archives/C01GTK53T8Q/p1628605049158100

**Review by TomatoToaster — CHANGES_REQUESTED** [review]
Left some comments but they seem NAB. Except the generated file one asked about it in slack.

File: android/app/src/main/java/com/expensify/chat/generated/BasePackageList.java
Comment (mountiny) [reply android/app/src/main/java/com/expensify/chat/generated/BasePackageList.java:13]: No, it was not meant to ge included, I added this accidently. I will remove it just in case!

**Comment by TomatoToaster** [comment]
>It is extremely sluggish which is as always I guess but then the gif of the animation does not move at all. So I do not know if that is just caused by the sluggishness of the system or the gif cant be animated on Android. Are you able to test it on physical device? I do not have any :/

I don't have a physical device to test on. Hmm instead of blocking on that though, can you add in the Tests/QA section to explicitly test this on and android device and to make note if it's frozen. Even if the gif freezes up just for android, I don't think it should block this from being merged. We can explore in a follow-up/regression how to fix that particular issue. Hopefully, it's just a quirk of the emulator.

Do you think that makes sense @mountiny, or do you have an idea what might be causing the frozen animation?

**Comment by mountiny** [comment]
@TomatoToaster That makes a lot of sense, I would not block it on this issue for sure. I will address the NAB comments and ask for final review.

I do not have a clue if this is just caused by the emulator or it really is like that on physical Android device (I hope the app is not so slow on physical android device as it is in the emulator 😅)

File: android/app/src/main/java/com/expensify/chat/generated/BasePackageList.java
Comment (TomatoToaster) [reply android/app/src/main/java/com/expensify/chat/generated/BasePackageList.java:13]: Yeah I think that's the move. If we decide on a solution for it in the thread, we can add it back or gitignore it in another PR.

File: src/pages/ReimbursementAccount/ReimbursementAccountPage.js
Comment (mountiny) [reply src/pages/ReimbursementAccount/ReimbursementAccountPage.js]: As this is indeed used only here, I will remove the `visible` parameter.

**Comment by mountiny** [comment]
@TomatoToaster I have updated the small changes, so asking for a final review. Thank you very much 🙌

**Review by TomatoToaster — APPROVED** [review]
LGTM, feel free to merge if you catch it passing the tests before I do.

**Comment by isagoico** [comment]
@vitHoracek Hello! We currently don't have a test domain to QA this since all our test domains already have the expensify card and the "Get started" won't show for us. Can this be tested internally?

**Comment by mountiny** [comment]
@isagoico Of course, sorry I have not added the label from the start! I will take care of it.

**Comment by MitchExpensify** [comment]
Tested - Looks great! Good to check off the list @isagoico

**Pull request merged**
